liqo-peering/templates/add-taint-label-to-virtualnode.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if eq .Values.role "consumer" }}
```

Comment:
It would be nice to add one/two lines of comment at the beginning of each yaml template to briefly explain what it does.